website/docs/reference/model-configs.md

```diff
@@ -2,10 +2,166 @@
 title: Model configurations
 ---
 
-<Alert type='danger'>
+## Related documentation
+* [Models](docs/building-a-dbt-project/building-models)
+* [`run` command](command-line-interface/run)
 
-These docs are a placeholder for a yet-to-be-written reference section.
+## Available configurations
+### Model-specific configurations
 
-Please refer to the [docs section](docs/docs/building-a-dbt-project/building-models/configuring-models.md) for current documentation.
+<Tabs
+  groupId="config-languages"
+  defaultValue="yaml"
+  values={[
+    { label: 'YAML', value: 'yaml', },
+    { label: 'Config block', value: 'config', },
+  ]
+}>
+<TabItem value="yaml">
 
-</Alert>
+<File name='dbt_project.yml'>
+
+```yaml
+models:
+  [<resource-path>](resource-path):
+    [materialized](materialized): <materialization_name>
+    [sql_header](sql_header): <string>
+
+```
+
+</File>
+
+</TabItem>
+
+
+<TabItem value="config">
+
+<File name='models/<model_name>.sql'>
+
+```jinja
+
+{{ config(
+    [materialized](materialized)="<materialization_name>",
+    [sql_header](sql_header)="<string>"
+) }}
+
+```
+
+</File>
+
+</TabItem>
+
+</Tabs>
+
+
+### General configurations
+
+<Tabs
+  groupId="config-languages"
+  defaultValue="yaml"
+  values={[
+    { label: 'YAML', value: 'yaml', },
+    { label: 'Config block', value: 'config', },
+  ]
+}>
+<TabItem value="yaml">
+
+<File name='dbt_project.yml'>
+
+```yaml
+models:
+  [<resource-path>](resource-path):
+    [enabled](enabled): true | false
+    [tags](tags): <string> | [<string>]
+    [pre-hook](pre-hook): <sql-statement> | [<sql-statement>]
+    [post-hook](post-hook): <sql-statement> | [<sql-statement>]
+    [database](resource-configs/database): <string>
+    [schema](resource-configs/schema): <string>
+    [alias](resource-configs/alias): <string>
```

Comment:
do you think we should include database-specific configs here too? I think that might get pretty cluttered up (even if we use tabs). Might just be a good idea to link out to the other db-specific docs around here

Reply:
Yah I definitely want to give the database-specific configs some love, but just left it out for this PR